firewood/src/db/proposal.rs

```diff
@@ -218,7 +218,7 @@ impl Proposal {
         revisions.base_revision = Arc::new(rev.into());
 
         // update the rolling window of root hashes
-        revisions.root_hashes.push_front(hash.clone());
```

Comment:
👍

Reply:
Clippy did that because  I made `TrieHash` `Copy`.